src/index.html

```diff
@@ -11,8 +11,9 @@
     <section id="homePage">
 
     </section>
+    <button id="tmp-popup-open" type="button">Hey, I'm a popup</button>
+    <!-- change secction name to popup - standard notation -->
     <section id="popUp">
```

Comment:
Maybe the form and the div for the popup can be here and then append dynamically the fetch from the API

Reply:
Actually, the popup is generated dynamically using `fetchPopup.js`.

Reply:
I mean, the popup is inserted into the `popUP` section 😄